### PR TITLE
Add new base template with navbar

### LINKED ISSUE #86: Missing navbar template
**Describe the solution you'd like**
Create a navbar using django templates, so that it can be inherited from in every page.

**Additional context**
We need a navbar.

### PR DESCRIPTION
**Pull request by nadavsu**
The base template divides a page into four main parts:
1. The title marked with {% block title %}
2. The navbar items marked with {% block navbar %}
3. The content marked with {% block content %}
4. The footer marked with {% block footer %}

It also contains the bootstrap script/library.
Every page inheriting this template will have a navigation bar:

navbar

Fixes #86

### CODE REVIEW

**Review by OrAsaf — APPROVED** [review]

File: main/templates/main/base_template.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <div class="content">
+        {% block content %}{% endblock %}
+      </div>
+    </main>
+  </body>
+
+  <!--FOOTER-->
+  <footer>
+    {% block footer %}{% endblock %}
+  </footer>
```
Comment (tamirho) [inline main/templates/main/base_template.html]: I think that footer tag should be inside a body tag. Correct me if I'm wrong.

**Comment by tamirho** [comment]
LGTM

**Review by DanielMy777 — COMMENTED** [review]
Nice looking navbar!

**Review by tamirho — APPROVED** [review]

File: main/templates/main/base_template.html (hunk 1 above)
Comment (simzacks) [reply main/templates/main/base_template.html]: I Agree.  The footer probably won't show up if it's out of the body.
See: https://www.w3schools.com/tags/tag_footer.asp

**Review by simzacks — CHANGES_REQUESTED** [review]

**Review by simzacks — APPROVED** [review]

**Review by Omeramsc — CHANGES_REQUESTED** [review]
This change doesn't pass the CI; it won't get reviewed or merged until it will.

**Comment by nadavsu** [comment]
Had to merge with roo.me/main, works now.

**Comment by AmitAharoni** [comment]
Thats great!

**Pull request merged**